native_lib/DDGSyncCrypto.c

```diff
@@ -166,15 +166,18 @@ DDGSyncCryptoResult ddgSyncSeal(
     unsigned char *message,
     unsigned long long messageLength) {
 
-    unsigned char output[crypto_box_SEALBYTES + messageLength];
+    unsigned char* output = (unsigned char*)malloc((crypto_box_SEALBYTES + messageLength) * sizeof(unsigned char));
```

Comment:
`malloc` can return a null pointer in failure cases (same as `ddgSyncSealOpen` below). Suggest checking the result of `malloc` via `output`:
```
if (!output)
    return DDGSYNCCRYPTO_SEAL_FAILED;
```

Reply:
Thanks @MarkIngramUK, changed it.